.env.example

```diff
@@ -34,7 +34,48 @@ SMTP_HOST=<SMTP_HOST>
 SMTP_MESSAGE_FROM=<SMTP_MESSAGE_FROM>
 SMTP_PORT=<SMTP_PORT>
 SMTP_SECURE=<SMTP_SECURE>
+
+DATASET_CREATION_VALIDATION_ENABLED=true
+DATASET_CREATION_VALIDATION_REGEX="^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$"
+
+ADMIN_GROUPS=""
+DELETE_GROUPS=""
 CREATE_DATASET_GROUPS="all"
 CREATE_DATASET_WITH_PID_GROUPS=""
-DATASET_CREATION_VALIDATION_ENABLED=true
-DATASET_CREATION_VALIDATION_REGEX="^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$"
\ No newline at end of file
+CREATE_DATASET_PRIVILEGED_GROUPS=""
+CREATE_JOB_GROUPS=""
+UPDATE_JOB_GROUPS=""
+SAMPLE_PRIVILEGED_GROUPS="sampleingestor"
+SAMPLE_GROUPS="group1"
+PROPOSAL_GROUPS=""
+
+ACCESS_GROUPS_GRAPHQL_ENABLED=true
+ACCESS_GROUP_SERVICE_TOKEN=""
+ACCESS_GROUP_SERVICE_API_URL=""
+ACCESS_GROUP_SERVICE_HANDLER=""
+ACCESS_GROUPS_STATIC_ENABLED=true
+ACCESS_GROUPS_OIDCPAYLOAD_ENABLED=true
+
+OIDC_USERINFO_MAPPING_FIELD_USERNAME="iss, sub"
+OIDC_USERINFO_MAPPING_FIELD_DISPLAYNAME="preferred_username"
+OIDC_USERINFO_MAPPING_FIELD_EMAIL="email"
+OIDC_USERINFO_MAPPING_FIELD_GROUP="groups"
+OIDC_USERQUERY_OPERATOR=<"or"|"and">
+OIDC_USERQUERY_FILTER="username:username, email:email"
+
+ELASTICSEARCH_ENABLED=<"yes"|"no">
+APP_PORT=3003
+STACK_VERSION="8.8.2"
+CLUSTER_NAME="es-cluster"
+MEM_LIMIT="4G"
+MONGODB_COLLECTION="Dataset"
+ES_MAX_RESULT=100000
+ES_FIELDS_LIMIT=400000
+ES_INDEX="dataset"
+ES_PORT=9200
+ES_HOST="https://localhost:9200"
+ES_USERNAME="elastic"
+ES_PASSWORD="duo-password"
+ES_REFRESH=<"wait_for"|"false">
```

Comment:
Should this be a boolean? It's a bit weird to have "false" as an enum value.

Reply:
Yes..It's indeed looking weird. Somehow we were using `yes|no` patterns for enable/disable flags before and I followed the pattern for consistency. Now we mixed `yes|no` and `true|false` for flags.
I will create an issue regarding the inconsistency, maybe we should replace all of the `yes|no` to `true|false`